src/components/PageNavLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+// Utilites
```

Comment:
remove all comments

Reply:
this comment is for me but I can delet it )